src/Commands/PluginCommand.php

```diff
@@ -0,0 +1,568 @@
+<?php
+
+/**
+ * The PluginCommand class manages Terminus plugins.
+ */
+
+namespace Pantheon\Terminus\Commands;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Exceptions\TerminusNotFoundException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Manage Terminus plugins.
+ *
+ * @package Pantheon\Terminus\Commands
+ */
+class PluginCommand extends TerminusCommand
+{
+
+    /**
+     * Install one or more Terminus plugins.
+     *
+     * @command plugin:install
+     * @aliases plugin:add
+     *
+     * @option project_list A comma delimited list of one or more plugin projects to install
+     *
+     * @usage <URL to plugin archive, Git or Packagist project 1>,[URL to plugin archive, Git or Packagist project 2],...
+     */
+    public function install($project_list = '')
```

Comment:
If you declare this `array $project_list`, then you will get an array of arguments, e.g. `terminus plugin install org/plugin1 org/plugin2`